app/controllers/application_controller.rb

```diff
@@ -11,12 +11,16 @@ class ApplicationController < ActionController::Base
 
 private
 
-  helper_method :gds_editor?, :active_navigation_item
+  helper_method :gds_editor?, :user_can_schedule?, :active_navigation_item
```

Comment:
It feels like this commit is doing two things. Restricting access to the scheduling function, and refactoring how change notes are generated. Can you split them, so that each commit does one thing.

Reply:
Whoops - they were originally two commits.